index-handlers/src/main/java/no/sikt/nva/nvi/index/model/document/Approval.java

```diff
@@ -27,6 +30,23 @@ public static Builder builder() {
         return new Builder();
     }
 
+    public BigDecimal getPointsForAffiliation(NviContributor nviContributor, NviOrganization affiliation) {
+        return points.creatorAffiliationPoints().stream()
+                   .filter(isEqual(affiliation))
+                   .filter(isNviContributor(nviContributor))
```

Comment:
I would rename isEqual to something else, cause CreatorAffiliationPoints can not be equal Organization, maybe something like matchesAffiliationId or hasAffiliationId? Same with isNviContributor.

Reply:
Good points! Will do